components/list/list-item.js

```diff
@@ -342,10 +344,34 @@ class ListItem extends RtlMixin(LitElement) {
 		}));
 	}
 
+	_handleBlurLink() {
+		this._focusingLink = false;
+	}
+
 	_handleCheckboxChange(e) {
 		this.setSelected(e.target.checked);
 	}
 
+	_handleFocusIn() {
```

Comment:
These will be super useful when we want to hide the dragger as per designs for this list item so yay!!! I was not sure how to do it but with this super easy.